<?xml version="1.0" encoding="UTF-8"?>
<launch>
  <!-- <param name="/free_gait/adapter_plugin/preview" value="free_gait::AdapterDummy" />
  <param name="/free_gait/adapter_ros_interface_plugin" value="free_gait::AdapterRosInterfaceDummy" />
  <node pkg="free_gait_ros" type="steps_test" name="steps_test_node"
        output="screen">
  </node> -->
  <param name = "/free_gait/robot_description"
         value = "$(find quadruped_model)/urdf/simpledog.urdf"
         />
  <param name="/free_gait/stop_execution_service" value="/free_gait/stop_execution_service" />
  <param name="/free_gait/pause_execution_service" value="/free_gait/pause_execution_service" />
  <param name="/use_gazebo" value="true" />
  <param name="/use_preview" value="false" />
  <param name="/kinematics_control" value="false" />
  <param name="/free_gait/adapter_plugin/gazebo" value="free_gait_ros/AdapterGazebo" />
  <param name="/free_gait/adapter_ros_interface_plugin/gazebo" value="free_gait_ros/AdapterRosInterfaceGazebo" />

  <param name="/use_terrian_map" value="false" />
  <param name="/step_height" value="0.15" />

  <node pkg="free_gait_ros"
    type="action_server_test_node"
    name="action_server_test_node"
    output="screen">
  </node>
  <!-- <rosparam file="$(find balance_controller)/config/controller_gains.yaml" command="load" /> -->
  <!-- <node pkg="rqt_free_gait_action" type="rqt_free_gait_action" name="rqt_free_gait_action"/>
  <node pkg="rqt_free_gait_monitor" type="rqt_free_gait_monitor" name="rqt_free_gait_monitor"/> -->
  <!-- <include file="$(find rqt_free_gait_monitor)/launch/rqt_free_gait_monitor.launch"/> -->
  <param name="free_gait/action_server" value="/free_gait/action_server" />
  <param name="/free_gait/preview_topic" value="/free_gait/preview_topic" />
  <!-- <include file="$(find rqt_free_gait_action)/launch/rqt_free_gait_action.launch"/> -->
  <node pkg="free_gait_action_loader" type="action_loader.py" name="free_gait_action_loader" output="screen"/>

</launch>
